packs/data/feat-effects.db/effect-ghost-wrangler.json

```diff
@@ -0,0 +1,52 @@
+{
+    "_id": "SiegLMJpVOGuoyWJ",
+    "img": "systems/pf2e/icons/spells/charming-touch.webp",
+    "name": "Effect: Ghost Wrangler",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Ghost Wrangler]{Ghost Wrangler}</p>\n<p>Your fists gain the effects of the ghost touch property rune.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "definition": [
+                    {
+                        "or": [
+                            "item:slug:fist",
+                            "item:slug:unarmed-attack"
```

Comment:
Do we want to check for unarmed category?

Reply:
`Until the beginning of your next turn, your fists gain the effects of the ghost touch property rune` it wouldn't affect non-fist strikes. This is almost certainly being too selective but that's on Paizo for using that wording.